app/javascript/utils/form/validations.js

```diff
@@ -124,4 +124,20 @@ validate.isValidDOB = (person, errors, isPrimaryApplicant = false) => {
   return errors
 }
 
+validate.isValidDateObject = (date, errors) => {
```

Comment:
hm this fn and fn above feel a bit copy-pasta-y. I wonder if there's a way we could make it a bit more DRY. Maybe just pass the date, and the field name to get to where you set the errors?